go.mod

```diff
@@ -1,18 +1,9 @@
-module github.com/go-playground/validator/v10
+module github.com/beautifulentropy/validator/v10
 
 go 1.18
 
 require (
-	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/go-playground/assert/v2 v2.2.0
 	github.com/go-playground/locales v0.14.1
 	github.com/go-playground/universal-translator v0.18.1
-	github.com/kr/pretty v0.3.0 // indirect
-	github.com/leodido/go-urn v1.2.1
```

Comment:
It looks like this is the only third-party dependency which is removed, and the other removals are transitively depended on by this package.

Why not just remove the one constraint which uses this dependency (`isUrnRFC2141`) and leave everything else as-is? Doing so would minimize the diff in this fork.

Reply:
I agree. The rest of the diff offers us very little except a slimmer codebase and could make merging upstream commits more difficult. I've gone ahead and removed `isURN` which introduces the bulk of these dependencies. I've also removed `eth_addr` and `bcp47_language_tag` which introduced newer versions of `golang.org/x/crypto v0.5.0` and `golang.org/x/text v0.6.0` than those currently required by Boulder.